tools/topology/CMakeLists.txt

```diff
@@ -122,6 +122,7 @@ set(TPLGS
 	"sof-tgl-max98357a-rt5682\;sof-tgl-max98357a-rt5682\;-DCODEC=MAX98357A\;-DFMT=s16le"
 	"sof-tgl-max98357a-rt5682\;sof-tgl-rt1011-rt5682\;-DCODEC=RT1011\;-DFMT=s24le"
 	"sof-tgl-max98373-rt5682\;sof-tgl-max98373-rt5682\;-DAMP_SSP=1"
+	"sof-tgl-max98373-rt5682\;sof-tgl-max98373-rt5682-xperi\;-DAMP_SSP=1\;-DXPERI=1"
```

Comment:
Please have a look at the format and of other commit messages to align the subject. The commit message should also say why you are making this change e.g.
```
topology: add support for xperi processing feature X
This patch adds IIR support as it is required to as a dependecy to enable xperi feature X....

Signed-off-by:....
```

Reply:
Updated commit message as described

Reply:
given we ware using generic components mind changing the flag name to something generic as well like "INCLUDE_EQ"?

Reply:
Yes I think this is a good idea - I'll update it. 
Generally what is the strategy for topologies in the long term. Surely as more devices are made that all target the same topology if there's different requirements in terms of the components/pipelines then the topology is going to become more complicated